omr/importpdf.cpp

```diff
@@ -275,8 +303,11 @@ Score::FileError importPdf(MasterScore* score, const QString& path)
 
       OmrState state;
       state.score = score;
-      foreach(OmrPage* omrPage, omr->pages())
-            state.importPdfPage(omrPage);
+          foreach(OmrPage* omrPage, omr->pages()){
```

Comment:
indentation is wrong.
You can use `for (OmrPage* ormPage : omr->pages()) {``
Note the space between ) and {